prepare/kotlin-compiler-internal-test-framework/build.gradle.kts

```diff
@@ -20,4 +20,8 @@ dependencies {
 publish()
 runtimeJar()
 sourcesJar()
-javadocJar()
\ No newline at end of file
+javadocJar()
+
+tasks.named("sourcesJar").configure {
```

Comment:
Is that what you were expecting @runningcode ?

The task is still eagerly created despite relying on the task configuration avoidance API
https://github.com/jprinet/kotlin/blob/3ee8aabe4cbae7ddc5175e85a440a9ae42677465/buildSrc/src/main/kotlin/tasks.kt#L237

Reply:
Thanks this looks good. It is likely some other piece of build logic which is eagerly configuring the task but at least we are not adding code which is causing eager configuration.

Small nit: since the `sourcesJar` task returns a task provider, you can directly call `configure` on that.